src/esp/sensor/Sensor.cpp

```diff
@@ -53,12 +56,33 @@ void Sensor::setTransformationFromSpec() {
   node().rotateZ(Magnum::Rad(spec_->orientation[2]));
 }
 
+gltensor::GLTensorParam::ptr Sensor::glTensorParam() const {
+  if (tgt_ == nullptr)
+    throw std::runtime_error("Sensor has no rendering target");
+
+  switch (spec_->sensorType) {
+    case SensorType::COLOR:
+      return tgt_->glTensorParamRgba();
+
```

Comment:
break; // even it is not necessary, it is good to have it to make the code less error-prone in the future.

Reply:
:thinking: I fear that may cause some compilers to complain about unreachable code, don't remember which tho.